DirectEmployers/MyJobs
# [PD-3683] Hide "Goal Minus -" from Users

## What's this PR do?
Instead of displaying Goal Minus One/Two, we are providing more context to those labels.

## Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/27963853/30755021-fd417706-9f92-11e7-8a78-b9cdc4c3c173.png)

![image](https://user-images.githubusercontent.com/27963853/30761395-159ddd30-9fac-11e7-8209-21f61ed572f8.png)


## Where should the reviewer start?
myreports/datasources/util.py L#424

## How should this be manually tested?
Go to "Goal Flow" in site traffic v2 report, drilldown and notice the first column is labelled correctly, you should no longer see "Goal Minus *" anywhere, including exported csv/excel

## PR Checklist

### Front-End


### Separation of Concerns
